examples/simple/outputs.tf

```diff
@@ -70,29 +70,20 @@ output "cluster_oidc_issuer_url" {
   value       = module.eks.cluster_oidc_issuer_url
 }
 
-# aws_iam_role.materialize_s3.arn
 output "materialize_s3_role_arn" {
   description = "The ARN of the IAM role for Materialize"
-  value       = aws_iam_role.materialize_s3.arn
-}
-
-output "operator_details" {
-  description = "Details of the installed Materialize operator"
-  value = var.install_materialize_operator ? {
-    namespace             = module.operator[0].operator_namespace
-    release_name          = module.operator[0].operator_release_name
-    release_status        = module.operator[0].operator_release_status
-    instances             = module.operator[0].materialize_instances
-    instance_resource_ids = module.operator[0].materialize_instance_resource_ids
-  } : null
+  value       = module.operator.materialize_s3_role_arn
 }
 
 output "nlb_details" {
   description = "Details of the Materialize instance NLBs."
   value = {
-    for nlb in module.nlb : nlb.instance_name => {
-      arn      = nlb.nlb_arn
-      dns_name = nlb.nlb_dns_name
-    }
+    arn      = try(module.materialize_instance.nlb_arn, null)
+    dns_name = try(module.materialize_instance.nlb_dns_name, null)
```

Comment:
So ...even though
```
module.materialize_nlb[0].aws_lb.nlb: Still creating... [2m40s elapsed]
module.materialize_nlb[0].aws_lb.nlb: Creation complete after 2m42s [id=arn:aws:elasticloadbalancing:us-east-1:400121260767:loadbalancer/net/kay-test/1ebf71a043aa1cf0]
```
the output has:
```
nlb_details = {
  "arn" = null
  "dns_name" = null
}

Reply:
Ah yes good catch about the nlb details, I pushed a change for this. Will look into the certificate authority data suggestion as well!